### PR TITLE
feat: websockets example

### PR DESCRIPTION
**Pull request by lino-levan**

### CODE REVIEW

File: data/websocket.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+// First we need to use the WebSocket constructor to initiate
+// our connection to an external server
+const socket = new WebSocket("ws://ws.example.com");
+
+// Before we do anything with the websocket, we should
+// wait to make sure that we are connected. We can do
+// so by listening to the "open" event.
+socket.addEventListener("open", () => {
+  // We can ready the "ready state" of our instance
+  // this determines if we are able to send messages.
```
Comment (kt3k) [inline data/websocket.ts:20]: nit:
```suggestion
  // We can read the "ready state" of our instance.
  // This determines if we are able to send messages.
```

File: data/websocket.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  // We can now send messages to the server. The messages
+  // can be of the type string, ArrayBuffer, Blob, or
+  // TypedArray. In this case we will be sending a string
+  socket.send("ping");
+});
+
+// We can handle messages back from the server by listing
+// to the "message" event. We can ready the data sent by
+// the server using the data property of the event.
```
Comment (kt3k) [inline data/websocket.ts:32]:
```suggestion
// We can handle messages back from the server by listening
// to the "message" event. We can read the data sent by
// the server using the data property of the event.
```

**Comment by lino-levan** [comment]
Thanks for the review

File: data/websocket.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
+ * @title Outbound Websockets
```
Comment (kt3k) [inline data/websocket.ts:2]:
```suggestion
 * @title Outbound WebSockets
```

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**